### PR TITLE
feat: added fields and 3 search queries - Streets, Districts, Regions…

### PR DESCRIPTION
**Pull request by MykhailoMe**
Implementation of GraphQL search queries based on existing (API) UA Addresses.

But there are two questions towards:
DistrictFilter, SettlementFilter - probably it is necessary to define other similar filters just for Settlement query and District query. Aka DistrictFilterMain, SettlementFilterMain, as DistrictFilter and SettlementFilter are used within other existing objects.
Please, let me know when you are ready to discuss.

Zube: https://zube.io/edenlabllc/e-health/c/10221
Apiary:
https://uaddressesapi.docs.apiary.io/#reference/0/search-districts/list-of-districts-by-search-params
https://ehealthmisapi1.docs.apiary.io/#reference/public.-medical-service-provider-integration-layer/search-streets/list-of-streets-by-search-params

### CODE REVIEW

File: __schema__/settlements.graphql [hunk 1]
```diff
@@ -1,9 +1,28 @@
+"""
+A condition to be used against `Settlement` object types. All fields are tested for equality and combined with a logical ‘and.’
+"""
+input SettlementFilterSubDistrict {
```
Comment (SergeyTatarenko) [inline __schema__/settlements.graphql:4]: For what purpose this separate filter needed?
Why not use "name" param in already existing SettlementFilter ?

File: __schema__/districts.graphql [hunk 2]
```diff
@@ -56,7 +70,7 @@ type District implements Node {
   "Reads and enables pagination through a set of `Settlement`."
   settlements(
     "A condition to be used in determining which values should be returned by the collection."
-    filter: SettlementFilter
+    filter: SettlementFilterSubDistrict
```
Comment (SergeyTatarenko) [inline __schema__/districts.graphql:73]: what purpose of separate filter?

File: __schema__/districts.graphql [hunk 3]
```diff
@@ -1,9 +1,23 @@
 """
 A condition to be used against `District` object types. All fields are tested for equality and combined with a logical ‘and.’
 """
+input DistrictFilterSubRegion {
```
Comment (SergeyTatarenko) [inline __schema__/districts.graphql:4]: What purpose of separate filter? Why "name" param in old DistrictFilter doesn't satisfy requirements?

File: __schema__/districts.graphql [hunk 4]
```diff
@@ -56,7 +70,7 @@ type District implements Node {
   "Reads and enables pagination through a set of `Settlement`."
   settlements(
```
Comment (SergeyTatarenko) [inline __schema__/districts.graphql:71]: I don't see this attr in real schema for a District node

File: __schema__/regions.graphql [hunk 5]
```diff
@@ -14,7 +56,7 @@ type Region implements Node {
   "Reads and enables pagination through a set of `District`."
   districts(
```
Comment (SergeyTatarenko) [inline __schema__/regions.graphql:57]: I don't see this attr in real schema

File: __schema__/regions.graphql [hunk 6]
```diff
@@ -14,7 +56,7 @@ type Region implements Node {
...
     "A condition to be used in determining which values should be returned by the collection."
-    filter: DistrictFilter
+    filter: DistrictFilterSubRegion
```
Comment (SergeyTatarenko) [inline __schema__/regions.graphql:59]: Why old filter doesn't satisfy requirements?

File: __schema__/streets.graphql [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  "Street type"
+  streetType: String!
+  "Settlement name"
+  settlementName: String!
+  "Date and time when record was inserted."
+  insertedAt: DateTime!
+  "Date and time when record was updated."
+  updatedAt: DateTime!
+
+}
```
Comment (SergeyTatarenko) [inline __schema__/streets.graphql:65]: new line in the end

File: __schema__/regions.graphql (hunk 6 above)
Comment (MykhailoMe) [reply __schema__/regions.graphql]: Removed

File: __schema__/regions.graphql (hunk 5 above)
Comment (MykhailoMe) [reply __schema__/regions.graphql]: You are right. Removed. There was an idea to simplify flow on FE.

File: __schema__/districts.graphql (hunk 4 above)
Comment (MykhailoMe) [reply __schema__/districts.graphql]: Removed.

File: __schema__/districts.graphql (hunk 2 above)
Comment (MykhailoMe) [reply __schema__/districts.graphql]: Removed.

File: __schema__/streets.graphql (hunk 7 above)
Comment (MykhailoMe) [reply __schema__/streets.graphql:65]: done.

**Review by SergeyTatarenko — APPROVED** [review]

**Pull request merged**